### PR TITLE
Osprey Remodel

### PR DESCRIPTION
**Pull request by Saugia**
**Content (Artwork)**

## Summary
OspreyComp

Along with all of the other new ship updates, the Osprey is a ship that's been in the backlines a little in terms of the model. Some details such as the guns and turret mounts were updated and look much better, but the state of the model itself is still a bit clunky. This PR focuses on an attempt to bring the Osprey up to snuff.

The biggest offender to the model was the nature of the clipping of all of the "feather plates". Although it's not something that can be told at first glance, a lot of them are clipping into each other, and there are some overlaps that are a little iffy. Using the same amount of plates, I've re-arranged them to settle in with each other a lot more cleanly while keeping the feathered look of the original. The result is a more armor-plated looking design where they seem more intentional to the design, rather than plates there just for looks. With that said, I made them less pointy, which adds more to the human-looking design. (The only other ship currently in human space with very pointed edges is the Corvette which is also dated.)

The nose and cockpit is also a bit too pointy- almost as if it's a Remnant ship- so I thickened that as well.

A common feature of Tarazed ships is inset details, which I applied to the plates in place of the empty holes on top of the originals. Together with the new organization, it is cleaner looking and results in way less unsightly clipping. There are also a few new structural details to the design in the same style as other Tarazed details.

The turrets are given their own dedicated mounts, and are now less swamped than they were before (especially no longer hiding almost underneath some of the plates). 

Overall, the approach is similar to the new models of the city ships- to make the Osprey look more structurally a ship, and less shapes shoved together. I understand that some people may still choose the "Cool Looks" of the original over the more "Structurally Sound Looks" of the newer model, so I did my best to keep an overall similar design.

Ospflying
OspreyCombat
## Save File
This save file can be used to play through the new mission content:
[Sherrol Mane Osprey.txt](https://github.com/endless-sky/endless-sky/files/8093638/Sherrol.Mane.Osprey.txt)
You can find Ospreys anywhere in FW space or by Tarazed.

**Edit: After some community feedback, I added a bit more color to the design:**

osprey0
osprey1

Also, as before, I do not intend on touching the pers ship unless I am asked to.

## PR Checklist
 - [x] I updated the copyright attributions, or decline to claim copyright of any assets produced or modified
 - [x] I uploaded the necessary image, blend, and texture assets here: (https://drive.google.com/drive/folders/1u50_Cf4UVfxYyQGOe3jyyeyIYyjEfCVt?usp=sharing)
 - [x] I created a PR to the [endless-sky-high-dpi repo](https://github.com/endless-sky/endless-sky-high-dpi) with the `@2x` versions of these art assets: (https://github.com/endless-sky/endless-sky-high-dpi/pull/211)

### CODE REVIEW

**Review by McloughlinGuy — APPROVED** [review]
Looks more like an actual ship now. I like it.

**Comment by EricD112** [comment]
This remodel looks gorgeous in my opinion.

**Review by Hurleveur — APPROVED** [review]
Looks great! I really like how most ships are getting an upgrade to the old designs, this one looks sharper, more precise whilst staying true to the original idea.

**Comment by EjoThims** [comment]
With the additional colors added on, definitely feels more right.

I can't help but think it's just a bit too bulky/beefy now though.

**Comment by Saugia** [comment]
> I can't help but think it's just a bit too bulky/beefy now though.

I know I have a habit of doing that on my reworks, but I would have to say this isn't necessarily a problem when it comes to the Osprey's model, and I would have to disagree. The reason it looks "bulkier" is partly by the new insets and is a slight optical illusion, and the fact that most fins are a bit flatter in rotation now, rather than rotated almost completely sideways into one another. 

Even though the Osprey is regarded as a lighter armor warship, the visual representation of the original already seems very breakable, so that's also a reason why I don't find the bulkiness too much of a problem.

Additionally, the turret mounts may be adding to this, which I think is fine because it's way better compared to before where they turrets were so pushed into the back that they looked like they would hit the ship itself. Although functionality isn't as important as "look cool" in a game like ES (sometimes), in my opinion it far suits the ship better than before this way.

**Review by Zitchas — APPROVED** [review]
I think this walks a pretty good line between "cool" and "looks like a functional spaceship." Likewise, it looks more like it could actually be built with human technology.

File: copyright [hunk 1]
```diff
@@ -1377,12 +1377,14 @@ Files:
...
  images/ship/mule*
+ images/ship/osprey*
  images/thumbnail/aerie*
  images/thumbnail/bactrian*
  images/thumbnail/mule*
+ images/thumbnail/osprey*
 Copyright: Saugia <https://github.com/Saugia>
 License: CC-BY-SA-4.0
-Comment: Derived from works by Michael Zahniser (under the same license).
+Comment: Derived from works by Michael Zahniser (under the same license) and detailed by Anarchist2.
```
Comment (Amazinite) [inline copyright:1387]: If this updated comment doesn't apply to all these images and is instead only for the Osprey then you should break the Osprey copyright off into its own section.

```
Files:
 images/ship/aerie*
 images/ship/bactrian*
 images/ship/mule*
 images/thumbnail/aerie*
 images/thumbnail/bactrian*
 images/thumbnail/mule*
Copyright: Saugia <https://github.com/Saugia>
License: CC-BY-SA-4.0
Comment: Derived from works by Michael Zahniser (under the same license).

Files:
 images/ship/osprey*
 images/thumbnail/osprey*
Copyright: Saugia <https://github.com/Saugia>
License: CC-BY-SA-4.0
Comment: Derived from works by Michael Zahniser (under the same license) and detailed by Anarchist2.
```

File: data/human/ships.txt [hunk 2]
```diff
@@ -2539,17 +2539,16 @@ ship "Osprey"
 		"Impala Plasma Thruster"
 		"Impala Plasma Steering"
 		"Hyperdrive"
-
```
Comment (Amazinite) [inline data/human/ships.txt:2542]: Not again. :(

**Review by Amazinite — CHANGES_REQUESTED** [review]

**Pull request merged**